net/rtpengine/patches/06-fix-iptables-extension-compilation.patch

```diff
@@ -0,0 +1,11 @@
+--- a/iptables-extension/libxt_RTPENGINE.c
++++ b/iptables-extension/libxt_RTPENGINE.c
```

Comment:
Where is the commit message and commit description?
 - Who is the author?
  - Is this patch taken from upstream? If not, are you going to upstream this?

Reply:
I sent it upstream:
https://github.com/sipwise/rtpengine/pull/1498

Reply:
Now, it's better thanks! Hopefully, we don't need to maintain this patch forever and we will see what the upstream says about it.